install.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
```

Comment:
I would add:
- `set -e` to force the script fail if something fails
- auto cd to the script folder, to avoid problems related with running the script from a different location